### ISSUE TITLE
Improving accessibility: Updating the MathJax version from 3.2.2 to 4.1.2

### ISSUE THREAD

**Issue by b-g-h** [marp-team/marp-cli#721]
- Problem: Currently, mathematical expressions in the HTML version cannot be read by a screen reader.
- Solution: The latest version of MathJax should be implemented, as it brings significant improvements to accessibility. Thank you for your work.

**Comment by yhatt** [comment]
Duplicated: MathJax is s a dependency of [Marp Core](https://github.com/marp-team/marp-core), not Marp CLI. Please track https://github.com/marp-team/marp-core/issues/405.

ref. https://github.com/marp-team/.github/blob/master/CONTRIBUTING.md
